cli/src/cluster_query.rs

```diff
@@ -2181,6 +2181,9 @@ pub fn process_calculate_rent(
     data_length: usize,
     use_lamports_unit: bool,
 ) -> ProcessResult {
+    if data_length > 10_000_000 {
+        println!("Warning: Account's maximum size is 10 MB".to_string());
```

Comment:
Nits for clarity:

* can you use `eprintln`? This way, if someone is parsing JSON output, they'll still be able to parse the output.
* can you give the amount provided? Ie. "Warning: ... provided {data_length}"
* wording update
* let's give the full byte amount to be clear

```suggestion
        eprintln!("Warning: Maximum account size is {MAX_PERMITTED_DATA_LENGTH} bytes, {data_length} provided");
```

Reply:
@joncinque thank you for the suggestion, updated the code